2021/yearbook.html

```diff
@@ -44,6 +44,12 @@ <h2>2021 Software Development Engineer Bootcamp</h2>
           <figcaption>Natasha</figcaption>
         </figure>
       </a>
+      <a href="https://thamimk.github.io/">
+        <figure>
+          <img src="images/Thami.jpg" alt="Thamsanqa" target="_blank" />
```

Comment:
The `target="_blank"` should be on the `a` tag (https://www.w3schools.com/tags/att_a_target.asp)

Reply:
Also Github says that this branch has conflicts that must be resolved, can you try to rebase the commits from `main` to your branch/repo or use Github Desktop to fix the conflicts ?
I will help if needed !

Reply:
I tried many times to resolve but a new error always comes up. 
I'll just leave it for now.